### PR TITLE
updated travis script by adding quotes

### PR DESCRIPTION
**Pull request by Mathadon**
this fixes an issue where not all unit tests are run
see https://github.com/lbl-srg/BuildingsPy/issues/208 and #928

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -46,7 +46,7 @@ before_install:
 install:
   - pip install --upgrade pip setuptools wheel
   - pip install --only-binary=numpy,scipy,matplotlib numpy scipy matplotlib
-  - pip install git+https://github.com/lbl-srg/BuildingsPy@master
```
Comment (mwetter) [inline .travis.yml]: Why is this change needed?

**Comment by Mathadon** [comment]
@mwetter I must have started from the wrong branch since this is indeed not intended.. sorry about that. I have reverted the change.

**Comment by mwetter** [comment]
@Mathadon No problem, good thing we do code review.
